### PR TITLE
test

### LINKED ISSUE #9: Vipin workflow
Vipin will choose an image, add a caption, and add both to a file.

- [ ] Create a branch
- [ ] Edit the file
- [ ] Commit the changes
- [ ] Create a Pull Request
- [ ] Request a Review
- [ ] Make more changes
- [ ] Get an approval
- [ ] Merge the Pull Request

creating issues
https://githubtraining.github.io/training-manual/GH4D/02_getting_started.html

### PR DESCRIPTION
**Pull request by vipinranamca**
Creating a pull request to change my caption. 
@michaelxluc modified funny

Closes #9

### CODE REVIEW

File: _posts/0000-01-13-vipinranamca.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![morpheus](https://cloud.githubusercontent.com/assets/16547949/25400958/410316ee-29c2-11e7-9e6d-9d13999e76e3.jpg)
 {: .center}
 
-Caption here
+I really miss Groot
```
Comment (vipinranamca) [inline _posts/0000-01-13-vipinranamca.md]: vipin here

Comment (vipinranamca) [reply _posts/0000-01-13-vipinranamca.md]: testing

**Review by vipinranamca — COMMENTED** [review]
testing

File: _posts/0000-01-13-vipinranamca.md (hunk 1 above)
Comment (vipinranamca) [inline _posts/0000-01-13-vipinranamca.md]: one more

**Review by michaelxluc — APPROVED** [review]
Looks good.

**Review by vipinranamca — COMMENTED** [review]
testing one more

**Pull request merged**